src/goose/console/views/redis/home.clj

```diff
@@ -0,0 +1,30 @@
+(ns goose.console.views.redis.home
+  (:require [goose.console.views.layout :as layout]))
+
+(defn header [_]
+  [:header
+   [:nav
+    [:div.nav-start
+     [:div.goose-logo
+      [:a {:href "/"}
+       [:img {:src "img/goose-logo.png" :alt "goose-logo"}]]]
+     [:div#app-name "AppName"]
+     [:div#menu
+      [:a {:href "/enqueued"} "Enqueued"]
+      [:a {:href "/scheduled"} "Scheduled"]
+      [:a {:href "/periodic"} "Periodic"]
+      [:a {:href "/batch"} "Batch"]
+      [:a {:href "/dead"} "Dead"]]]]])
```

Comment:
note: The toggle button for theme change is not present in the navbar currently. It will be added when the story to "change theme" is picked.